.github/workflows/ci.generate.ts

```diff
@@ -688,6 +688,16 @@ const ci = {
           ].join("\n"),
           env: { CARGO_PROFILE_DEV_DEBUG: 0 },
         },
+        // Uncomment for remote debugging
+        // {
+        //   name: "Setup tmate session",
+        //   if: [
+        //     "(matrix.job == 'test' || matrix.job == 'bench') &&",
+        //     "matrix.profile == 'release' && (matrix.use_sysroot ||",
+        //     "github.repository == 'denoland/deno')",
+        //   ].join("\n"),
+        //   uses: "mxschmitt/action-tmate@v3",
+        // },
```

Comment:
Is this left on purpose?

Reply:
yes, ive used it quite a few times for segfaults in CIs